projects/core/src/cms/model/componentIds.model.ts

```diff
@@ -0,0 +1,3 @@
+export interface ComponentIds {
```

Comment:
For `get`, we don't need this model. But, for `post` we need. 'post' requires ComponentIDListWsDTO, which is {idList: string[];}. So, the IdList model you removed is for this purpose.